packages/backend/server/routes/stats.get.ts

```diff
@@ -1,40 +1,23 @@
-import { sha256 } from "ohash"
+import { sha256 } from "ohash";
 
 export default eventHandler(async (event) => {
   try {
     const binding = useBinding(event);
 
-    let started = false
-
-    let cursor: string | undefined;
+    const query = getQuery(event);
 
+    let cursor = query.cursor || undefined;
     let objectCount = 0;
 
     const encoder = new TextEncoder();
-
+    
     const packagesPrefix = `${usePackagesBucket.base}:`;
     const cursorsPrefix = `${useCursorsBucket.base}:`;
 
     const stream = new ReadableStream({

```

Comment:
now we do not have any use for readable stream, let's remove it!

Reply:
consider it done